homeassistant/components/ambee/const.py

```diff
@@ -38,175 +34,202 @@
-            ATTR_ICON: "mdi:tree",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "tree_hazel": {
-            ATTR_NAME: "Hazel Tree Pollen",
-            ATTR_ICON: "mdi:tree",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "tree_oak": {
-            ATTR_NAME: "Oak Tree Pollen",
-            ATTR_ICON: "mdi:tree",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "tree_pine": {
-            ATTR_NAME: "Pine Tree Pollen",
-            ATTR_ICON: "mdi:tree",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "tree_plane": {
-            ATTR_NAME: "Plane Tree Pollen",
-            ATTR_ICON: "mdi:tree",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "tree_poplar": {
-            ATTR_NAME: "Poplar Tree Pollen",
-            ATTR_ICON: "mdi:tree",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "weed_chenopod": {
-            ATTR_NAME: "Chenopod Weed Pollen",
-            ATTR_ICON: "mdi:sprout",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "weed_mugwort": {
-            ATTR_NAME: "Mugwort Weed Pollen",
-            ATTR_ICON: "mdi:sprout",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "weed_nettle": {
-            ATTR_NAME: "Nettle Weed Pollen",
-            ATTR_ICON: "mdi:sprout",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-        "weed_ragweed": {
-            ATTR_NAME: "Ragweed Weed Pollen",
-            ATTR_ICON: "mdi:sprout",
-            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-            ATTR_UNIT_OF_MEASUREMENT: CONCENTRATION_PARTS_PER_CUBIC_METER,
-            ATTR_ENABLED_BY_DEFAULT: False,
-        },
-    },
+SENSORS: dict[str, list[SensorEntityDescription]] = {
+    SERVICE_AIR_QUALITY: [
```

Comment:
This could be a tuple :)

Reply:
And the list a set :)

Reply:
![image](https://user-images.githubusercontent.com/195327/126700643-ff0c631d-f787-4472-94fd-be3519bbd225.png)

If you don't mind, I'm skipping that route :)